packages/compass-e2e-tests/helpers/compass.js

```diff
@@ -303,6 +305,37 @@ function addDebugger(app) {
   }
 }
 
+function wrapCommands(app) {
+  const proto = Object.getPrototypeOf(app.client);
+  const commands = Object.keys(proto).filter((key) => {
+    return (typeof proto[key] === 'function' && !key.includes('.'));
+  });
+
+  const wrapped = {};
+  for (const command of commands) {
+    wrapped[command] = async function (...args) {
+      console.log(`${command} ${args.join(', ')}`);
+      const stack = (new Error(command)).stack;
+      try {
+        return await app.client[command].call(app.client, ...args);
+      } catch (error) {
+        // Log how we got here, but still throw the original error
+        error.stack = `${error.stack}\nvia ${stripWrapped(stack)}`;
```

Comment:
webdriver's errors lose basically all context. ie.

```
  Error: element ("[data-test-id="collection-header-title"][title="test.numbers"") still not visible after 5000ms
      at new WaitUntilTimeoutError (/Users/leroux.bodenstein/mongo/compass/node_modules/webdriverio/build/lib/utils/ErrorHandler.js:149:12)
      at /Users/leroux.bodenstein/mongo/compass/node_modules/webdriverio/build/lib/commands/waitUntil.js:29:19
```

This line gives you this for the same error:
```
  Error: element ("[data-test-id="collection-header-title"][title="test.numbers"") still not visible after 5000ms
      at new WaitUntilTimeoutError (/Users/leroux.bodenstein/mongo/compass/node_modules/webdriverio/build/lib/utils/ErrorHandler.js:149:12)
      at /Users/leroux.bodenstein/mongo/compass/node_modules/webdriverio/build/lib/commands/waitUntil.js:29:19
  via Error: waitForVisible
      at navigateToCollection (helpers/commands/navigate-to-collection-tab.js:22:18)
      at processTicksAndRejections (node:internal/process/task_queues:96:5)
      at async Object.navigateToCollectionTab (helpers/commands/navigate-to-collection-tab.js:29:5)
  via Error: navigateToCollectionTab
      at Context.<anonymous> (tests/ttfq.test.js:22:18)
      at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

Which is maybe a bit unorthodox, but really helps you see _which_ line caused the "WaitUntilTimeoutError" or whatever.

Reply:
Those node:internal lines and things from node_modules should be stripped now. Just helpers and tests remaining.